### PR TITLE
Apr 2041 gw clarification template

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.RoatpGateway.Domain/Apply/ApplyGatewayDetails.cs [hunk 1]
```diff
@@ -7,6 +7,5 @@ public class ApplyGatewayDetails
         public DateTime? SourcesCheckedOn { get; set; }
         public string Comments { get; set; }
         public DateTime? OutcomeDateTime { get; set; }
-        public DateTime? ClarificationRequestedOn { get; set; }
```
Comment (martin-hornsby) [inline src/SFA.DAS.RoatpGateway.Domain/Apply/ApplyGatewayDetails.cs:10]: Is this not being used anymore?

File: src/SFA.DAS.RoatpGateway.Web/ViewModels/RoatpGatewayPageViewModel.cs [hunk 2]
```diff
@@ -35,5 +35,8 @@ public class RoatpGatewayPageViewModel
         public string Heading { get; set; }
         public string Caption { get; set; }
         public string NoSelectionErrorMessage { get; set; }
+    
+        public string ClarificationAnswer { get; set; }
+        public DateTime? DateOfClarificationCheck { get; set; }
```
Comment (martin-hornsby) [inline src/SFA.DAS.RoatpGateway.Web/ViewModels/RoatpGatewayPageViewModel.cs:40]: not sure about naming. Isn't this when Clarification is Requested, or is it something else (like when it was recieved)?

File: src/SFA.DAS.RoatpGateway.sln [hunk 3]
```diff
@@ -7,7 +7,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.RoatpGateway.Web.Un
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.RoatpGateway.Web", "SFA.DAS.RoatpGateway.Web\SFA.DAS.RoatpGateway.Web.csproj", "{1A5F2493-2E43-414B-BB7A-729C31CA44C1}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.RoatpGateway.Domain", "SFA.DAS.RoatpGateway.Domain\SFA.DAS.RoatpGateway.Domain.csproj", "{6FB5B798-1C06-4F90-B663-14230D7B5DBB}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.RoatpGateway.Domain", "SFA.DAS.RoatpGateway.Domain\SFA.DAS.RoatpGateway.Domain.csproj", "{6FB5B798-1C06-4F90-B663-14230D7B5DBB}"
```
Comment (martin-hornsby) [inline src/SFA.DAS.RoatpGateway.sln:10]: The project type has changed to a web application, it should be changed back

**Review by martin-hornsby — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.RoatpGateway.Domain/Apply/ApplyGatewayDetails.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.RoatpGateway.Domain/Apply/ApplyGatewayDetails.cs:10]: I figured the new audit stuff would be enough, as that tracks the status change and the updatedon.  Not sure now though.  I've raised this with you in slack channel dev

File: src/SFA.DAS.RoatpGateway.Web/ViewModels/RoatpGatewayPageViewModel.cs (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.RoatpGateway.Web/ViewModels/RoatpGatewayPageViewModel.cs:40]: that's a leftover from refactoring, I'm removing it

File: src/SFA.DAS.RoatpGateway.Domain/Apply/ApplyGatewayDetails.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.RoatpGateway.Domain/Apply/ApplyGatewayDetails.cs:10]: Actually all I needed was the updatedon, which is already passed back as the field 'OutcomeMadeOn' from apply, so I didn't need it.  Re-requesting a review, as I've addressed all other issues.

Comment (martin-hornsby) [reply src/SFA.DAS.RoatpGateway.Domain/Apply/ApplyGatewayDetails.cs:10]: Cool

**Review by martin-hornsby — APPROVED** [review]

**Review by martin-hornsby — APPROVED** [review]

**Pull request merged**